src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (invertedRobot[value] !== undefined) {
+      return null;
+    }
```

Comment:
Nice work!
Just one little thing to change
Please see checklist item No.2. It would be better to check for duplicates using hasOwnProperty() method
![js-checklist-inverseRobot](https://github.com/mate-academy/js_inverse_robot/assets/134736240/7535d354-b0fc-4de6-9782-60a3c236a681)

Reply:
ty for your help !